python/pyarrow/_dataset_parquet.pyx

```diff
@@ -78,7 +239,8 @@ cdef class ParquetFileFormat(FileFormat):
         CParquetFileFormat* parquet_format
 
     def __init__(self, read_options=None,
-                 default_fragment_scan_options=None, **kwargs):
+                 default_fragment_scan_options=None,
+                 **kwargs):
```

Comment:
Small general comment: while there is nothing wrong with this change, trying to avoid such stylistic changes (or other whitespace changes) does make it easier to review the diff.